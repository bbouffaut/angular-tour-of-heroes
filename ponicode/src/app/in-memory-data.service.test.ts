import * as in_memory_data_service from "../../../src/app/in-memory-data.service"
// @ponicode
describe("createDb", () => {
    let inst: any

    beforeEach(() => {
        inst = new in_memory_data_service.InMemoryDataService()
    })

    test("0", () => {
        let result: any = inst.createDb()
        expect(result).toMatchSnapshot()
    })
})
