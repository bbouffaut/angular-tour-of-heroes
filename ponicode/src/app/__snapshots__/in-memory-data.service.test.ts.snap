// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createDb 0 1`] = `
Object {
  "heroes": Array [
    Object {
      "id": 11,
      "name": "Mr. Nice",
    },
    Object {
      "id": 12,
      "name": "Narco",
    },
    Object {
      "id": 13,
      "name": "Bombasto",
    },
    Object {
      "id": 14,
      "name": "Celeritas",
    },
    Object {
      "id": 15,
      "name": "Magneta",
    },
    Object {
      "id": 16,
      "name": "RubberMan",
    },
    Object {
      "id": 17,
      "name": "Dynama",
    },
    Object {
      "id": 18,
      "name": "Dr IQ",
    },
    Object {
      "id": 19,
      "name": "Magma",
    },
    Object {
      "id": 20,
      "name": "Tornado",
    },
  ],
}
`;
